authors/apps/authentication/serializers.py

```diff
@@ -43,7 +43,7 @@ class Meta:
         model = User
         # List all of the fields that could possibly be included in a request
         # or response, including fields specified explicitly above.
-        fields = ['email', 'username', 'password']
+        fields = ['id', 'email', 'username', 'password', 'is_active']
```

Comment:
Remove id and is_active as a return value.

Reply:
Can we have the password field as write_only like so:

class Meta:
        model = ModelA
        write_only_fields = ('password',)